modules/ROOT/pages/design-ghs-install-authorize.adoc

```diff
@@ -78,3 +78,17 @@ If you encounter errors when attempting to authorize the GitHub Synchronization
 . xref:authorize-the-github-synchronization-application-in-api-designer[Authorize the GitHub Synchronization Application].
 
 If you see an empty list, ensure you've installed the application either in a GitHub organization or in a GitHub user account before performing any sync action.
+
+== Disabling the GitHub Integration
+
+Your organization administrator can disable the Github Integration:
+
+. On the Design Center *Projects* page, click *Settings*.
+. Under *Integrations*, disable the Github Integration.
```

Comment:
This may be correct, but it is unclear if there is an action in this step. Do they need to click integrations?